python/cuml/benchmark/algorithms.py

```diff
@@ -263,6 +281,17 @@ def all_algorithms():
             accepts_labels=True,
             accuracy_function=cuml.metrics.accuracy_score,
         ),
+        AlgorithmPair(
```

Comment:
Suppose I would also like to set up a benchmark for sparse trees. Do I need to just add an `AlgorithmPair` with different parameters (suppose there are parameters to use sparse trees in FIL) and a different name?

Reply:
Yes, I believe so